float64[7] q_d
float64[7] dq_d
float64[7] ddq_d
